docs/contribute/dev-setup.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Dev setup'
+---
+
+To fix a bug or add a feature in [Storybook's monorepo](https://github.com/storybookjs/storybook), the first step is to set up your development environment.
+
+## Initial setup
+
+The following command downloads and builds the most recent development version of Storybook on your local machine:
```

Comment:
Do we just assume people know how to fork a repo?

Reply:
I think so. Table stakes, right?

Reply:
Indeed!